sec/intro.tex

```diff
@@ -7,56 +7,59 @@ \section{Introduction}\label{sec:intro}
 research data directly. That article is one of many in an increasing body of
 publications over the last decade that concern themselves with the state of
 research data~\cite{Higman2019}, its utilisation and
-openness~\cite{Aslam2017,Zuiderwijk2020}, and the importance of good
+openness~\cite{Aslam2017,Zuiderwijk2020}, and the importance of best
 practices~\cite{Colavizza2020,Corti2019}. These publications, for the most part,
 do not cater specifically to the increasingly popular concepts attached to
-\emph{Big Data}, where the volume and variety of a data source should be
+\emph{Big Data}, where the volume and variety of a data source should be  % TODO I dont fully understand what we're saying here.
 maximised~\cite{Batisti2019}. Instead, the focal point is in the quality of
 research data as a resource, and as a mark of the associated research's
 quality.~\cite{Stall2019} argues for the widespread use of a concept adopted
 already in the geosciences, that research data should be \emph{Findable,
 Accessible, Interoperable and Reusable} (FAIR) --- an acronym coined
 in~\cite{Wilkinson2016}.
+% TODO Can we mention an analogy to software here as well? (Briefly)
 
 In tandem with this shift was the rise in the use of software to implement and
 compute algorithms (including clustering), and so electronic data became
 essential. As the size and complexity of research data increased, so did the
-span of the field \emph{machine learning}. Machine learning is a loose term to
+span of the field \emph{machine learning}. Machine learning is a term to
 describe how a computer (machine) may learn (by use of statistics) from a
 potentially large source of data, without following explicit instructions. Owing
 in part to this broad definition, machine learning comprises a great deal of
 techniques that were borne out of statistics, including regression,
-classification and clustering, despite having lost its connection with the
-statistics that underpin it, at least colloquially.
+classification and clustering.
 
 Healthcare modelling is one crucial (albeit broad) branch of research in which
 decisions are increasingly being informed by data-driven
 methodologies~\cite{Alexander2018,Belle2015,RiosZertuche2020}, often based
 around machine learning techniques. Clustering is a long-standing and often-used
+% TODO Add a catch all reference for clustering?
```

Comment:
I think a catch-all reference for clustering in healthcare would be better placed here. What are your thoughts?